src/docfx/lib/git/models/GitCommit.cs

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+namespace Microsoft.Docs
+{
+    /// <summary>
+    /// The git commit information
+    /// </summary>
+    public class GitCommit
+    {
+        /// <summary>
+        /// Gets or sets the git commmit author name
+        /// </summary>
+        public string AuthorName { get; set; }
```

Comment:
>AuthorName [](start = 22, length = 10)

Should it contain both `Committer` and `Author`?

Reply:
what will be used finally?

---
In reply to: [180009486](https://github.com/dotnet/docfx/pull/2601#discussion_r180009486) [](ancestors = 180009486)